### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by Lqm26** [Yan98/Event-Camera-Data-Pre-training#5]
Thank you very much for your work. We have a question about the loss in the code. Line 89 of **pr_trainer.py** uses **loss_event + loss_image + kl_loss +cls_loss**. We would like to ask what the **cls_loss** is and why it is added. The paper only says that **loss_event + loss_image + kl_loss** is used.

**Comment by Yan98** [comment]
Thank you for your interest. Please refer to [here](https://github.com/Yan98/Event-Camera-Data-Pre-training/blob/main/model/hub_model.py#L194). cls_loss is used to monitor pre-training only, and gradients from labels to the backbone are prevented by detach.

**Comment by Lqm26** [comment]
Thanks for your answer. I have another question. Why do you need to perform the **(data["event1"] - 0.5) * 2** operation on the event in lines **64 and 65 of pr_trainer.py**?

**Comment by Yan98** [comment]
There is no specific reason for this, and it should also work if you do not perform the scaling.
